system-container/run-system-container.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+export ANSIBLE_LOG_PATH=/var/log/ansible.log
+exec ansible-playbook -i /etc/ansible/hosts ${OPTS} ${PLAYBOOK_FILE}
```

Comment:
`/etc/passwd` is group-writable by GID 0, and that trick allows `/usr/libexec/s2i/run` to work... it's still unclear to me why we need a different entry point, or if we need changes/fixes in playbook2image related to the entry point.